src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -119,31 +121,41 @@ export default {
 			enableLoanCardExp: false,
 			spotlightIndex: 0,
 			spotlightViewportObserver: null,
+			enableRelendingExp: false
 		};
 	},
 	apollo: {
 		query: userInfoQuery,
 		preFetch(config, client) {
-			return Promise.all([
-				client.query({ query: experimentAssignmentQuery, variables: { id: EXP_KEY } }),
-				client.query({ query: experimentAssignmentQuery, variables: { id: LOAN_CARD_EXP_KEY } }),
-				client.query({ query: experimentAssignmentQuery, variables: { id: CATEGORIES_REDIRECT_EXP_KEY } }),
-				client.query({ query: experimentAssignmentQuery, variables: { id: FIVE_DOLLARS_NOTES_EXP } }),
-				client.query({ query: experimentAssignmentQuery, variables: { id: FLSS_ONGOING_EXP_KEY } }),
-			]).then(() => {
-				const userInfoPromise = client.query({
-					query: userInfoQuery,
-				});
-
-				const recommendedLoansPromise = client.query({
-					query: flssLoansQueryExtended,
-					variables: prefetchedRecommendedLoansVariables
-				});
+			return client.query({
```

Comment:
Seeing this implemented, I think we can remove this first query and call `experimentAssignmentQuery` for the new experiment regardless of whether the user has balance. So this code can be removed, and the new `experimentAssignmentQuery` added below.